### PR TITLE
Adding Discord and Charger Connect to main page

### PR DESCRIPTION
**Pull request by jestes15**
# Adding Discord and Charger Connect to main page
Added a discord box with the discord link to the main page and redirected the event button on the main page to charger connect. Fixed the UI and site map for mobile users.

# Category
- [x] Bug Fix - Bug #27

### CODE REVIEW

**Comment by computablee** [comment]
Could we extract the binary search JS and all the other duplicated code into a .js and just import that? I see a lot of duplication and bringing it out into a single .js file will help with maintenance in the future.

File: aboutus.html [hunk 1]
```diff
@@ -84,7 +85,7 @@ <h1>Sierra Laney<br>President/Chair</h1>
                         <h2>Computer Science Major</h2>
                     </div>
                 </div>
-                <a class="button button-primary" href="http://livartly.com">PRESIDENT</a>
+                <a class="button button-primary" href="404.html">PRESIDENT</a>
```
Comment (Jedimaster4559) [inline aboutus.html]: Couldn't you just removed the href?

File: index.html [hunk 2]
```diff
@@ -84,9 +86,9 @@ <h2 style="font-size: large">Learn about what we can do for you!</h2>
                     <div class="rightimage">
                         <img src="images/philteaching.jpg" width="100%" alt="UAH ACM Nasa Space Apps 2018" />
                         <h1 style="font-size: large">CHECK OUT EVENTS</h1>
-                        <h2 style="font-size: large">Professional development and hackathons!</h2>
+                        <h2 style="font-size: large">Check out event we will be hosting with various speakers on different topics!</h2>
```
Comment (Jedimaster4559) [inline index.html]:
```suggestion
                        <h2 style="font-size: large">Check out events we will be hosting with various speakers on different topics!</h2>
```

File: scripts/resize-functions.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
+const binarySearch = dim => function bin(start, end) {
```
Comment (Jedimaster4559) [inline scripts/resize-functions.js:2]: Did you actually write this? If not, please cite your sources.

**Review by Jedimaster4559 — COMMENTED** [review]
See thoughts attached.

TBH, the more I read reviews for updates to the main site, the more I wish we we has some type of templates capability so we weren't re-writing this header on every page...

File: aboutus.html (hunk 1 above)
Comment (Jedimaster4559) [reply aboutus.html]: I realize that might cause some styling issues, but I think a 404 redirect here sends the wrong message to the user. If I get a 404, I think, "Oh, that page is somewhere but the link is broken, or it's been deleted, or something". This is a bit of a different case. Here its more like "That object shouldn't be clickable".

It's my opinion that you should never intentionally give the user a 404. A 404 just gives you a safe path out of any potential edge cases or unexpected user behaviour.

Comment (jestes15) [reply aboutus.html]: I realize that now, but I have no idea where to redirect to. I could make it non clickable though.

File: scripts/resize-functions.js (hunk 3 above)
Comment (jestes15) [reply scripts/resize-functions.js:2]: Thanks for reminding me about the binary search.

**Review by computablee — DISMISSED** [review]
This looks good to me

**Review by hestes835 — APPROVED** [review]
I see no issues.

**Review by sel3450 — APPROVED** [review]
approved

**Pull request merged**